bigbluebutton-html5/imports/api/2.0/annotation/index.js

```diff
@@ -0,0 +1 @@
+export default new Mongo.Collection('annotation');
```

Comment:
This really should be plural, `annotations`.
And the folder name - `2.0/annotations`
Just to be consistent with the rest of the apis